stock_picking_report_undelivered_product/models/stock_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    is_splitted = fields.Boolean(
```

Comment:
Shouldn't this functionallity go in another module ?

Reply:
Uppss!! I do not create other module for that...